### PR TITLE
Adds two new BR maps, basic mobs, BR tweaks & fixes

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Title. Adds two new maps: 
- Grasslands Hunt (peaceful)
- Meta Central (easy)

These maps add a new basic revolutionary mob and significantly upgrades the ai of basic deer.
This fixes an issue where modular maps were not correctly spawning mobs or adding them to the "mutable candidates" for antagonists.
There's also some balance changes to bitrunning vendor prices, which are generally now lower. This change is unrelated to the PR as a whole so I'm okay with removing it if there's concern

### photos
<details>
<summary>expand</summary>

Screenshot 2024-07-26 151822

Screenshot 2024-07-26 151834

</details>

### todo

- [x] Fix the revolutionary death anim
- [x] Make deer run when injured
## Why It's Good For The Game
New maps as a general positive for bitrunning
Bug fixes
Makes vendor choices for bitrunning-exclusive items generally less of a chore to get
## Changelog
jlsnow301, MMMiracles, KikoWen0, Ben10Omintrix
:cl:
add: Added two new bitrunning maps: Grasslands Hunt and Meta Central.
add: Deer are now more complex animals, granting them enhanced ability to run amok and chew your favorite plants.
balance: Reduced the cost of most BR vendor items.
fix: Fixes an issue where modular virtual domains spawned less mobs than intended.
fix: These modular spawns are now valid mutation targets to become an antagonist.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
let ghosts become a tider rev if they click on them

**Comment by jlsnow301** [comment]
> let ghosts become a tider rev if they click on them

i would need to bump this up from an easy domain, and i really want more easy domains :(

**Comment by Ghommie** [comment]
Make sure to get rid of wallening-related map issues with Meta-central.

File: code/modules/bitrunning/orders/tech.dm
Comment (Ghommie) [inline code/modules/bitrunning/orders/tech.dm:1]: if you're reducing the overall cost of tech, do it for the flip chip too (perhaps more than the rest idk).

File: code/modules/bitrunning/virtual_domain/modular_mob_segment.dm [hunk 1]
```diff
@@ -3,27 +3,49 @@
...
+	var/list/mobs = list()
+	/// Spawn no more than this amount
+	var/max = 4
 	/// Chance this will spawn (1 - 100)
 	var/probability = SPAWN_LIKELY
 
+
 /// Spawns mobs in a circle around the location
-/datum/modular_mob_segment/proc/spawn_mobs(turf/origin)
+/datum/modular_mob_segment/proc/spawn_mobs(turf/origin) as /list
```
Comment (Ghommie) [inline code/modules/bitrunning/virtual_domain/modular_mob_segment.dm:40]: Do we need the `as /list` parameter here?

File: code/modules/mob/living/basic/revolutionary.dm [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	var/obj/item/weapon_of_choice = pick(possible_weapons)
+	attack_sound = weapon_of_choice::hitsound
+	attack_verb_simple = possible_weapons[weapon_of_choice]
+	attack_verb_continuous = "[attack_verb_simple]s"
+
+	var/static/list/death_loot = list(/obj/effect/mob_spawn/corpse/human/revolutionary)
+	AddElement(/datum/element/death_drops, death_loot)
+	apply_dynamic_human_appearance(src, mob_spawn_path = /obj/effect/mob_spawn/corpse/human/revolutionary, l_hand = weapon_of_choice)
+
+	var/gender = pick("male", "female", "enby")
```
Comment (Ghommie) [inline code/modules/mob/living/basic/revolutionary.dm:93]:
```suggestion
	gender = pick(MALE, FEMALE, PLURAL)
```

File: code/modules/mob/living/basic/revolutionary.dm [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	var/first_name
+	switch(gender)
+		if("male")
```
Comment (Ghommie) [inline code/modules/mob/living/basic/revolutionary.dm:96]:
```suggestion
		if(MALE)
```

File: code/modules/mob/living/basic/revolutionary.dm [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+			first_name = pick(GLOB.first_names_male)
+			death_sound = pick(male_screams + monkey_screeches)
+		if("female")
```
Comment (Ghommie) [inline code/modules/mob/living/basic/revolutionary.dm:99]:
```suggestion
		if(FEMALE)
```

File: code/modules/mob/living/basic/revolutionary.dm [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+			first_name = pick(GLOB.first_names_female)
+			death_sound = pick(female_screams + monkey_screeches)
+		if("enby")
```
Comment (Ghommie) [inline code/modules/mob/living/basic/revolutionary.dm:102]:
```suggestion
		if(PLURAL)
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/bitrunning/virtual_domain/modular_mob_segment.dm (hunk 1 above)
Comment (jlsnow301) [reply code/modules/bitrunning/virtual_domain/modular_mob_segment.dm:40]: it conveys no benefit, no

File: code/modules/bitrunning/orders/tech.dm
Comment (jlsnow301) [reply code/modules/bitrunning/orders/tech.dm:1]: done

**Pull request merged**